contrib/inventory/ec2.py

```diff
@@ -267,6 +267,7 @@ def read_settings(self):
 
         # Route53
         self.route53_enabled = config.getboolean('ec2', 'route53')
+        self.route53_hostnames = config.get('ec2', 'route53_hostnames')
```

Comment:
Good point, important this remain compatible with old ini files. Have addressed as suggested.